packages/sqs-json-body-parser/__tests__/index.js

```diff
@@ -19,8 +19,8 @@ test.afterEach((t) => {
 test.serial('parses each body payload', async (t) => {
   const request = { event }
   const bodys = [{ one: 1 }, { two: 2 }, { three: 3 }]
-  event.Records.push({ ...event.Records[0] })
-  event.Records.push({ ...event.Records[0] })
+  event.Records.push(JSON.parse(JSON.stringify(event.Records[0])))
+  event.Records.push(JSON.parse(JSON.stringify(event.Records[0])))
```

Comment:
is this to do a deep copy of the objects?

Reply:
yes, it was a regression.

Reply:
* and to prevent mutations of nested objects